internal/pkg/cli/app_init.go

```diff
@@ -209,30 +209,30 @@ func (opts *InitAppOpts) listDockerfiles() ([]string, error) {
 	if err != nil {
 		return nil, fmt.Errorf("read directory: %w", err)
 	}
-	var dockerfiles []string
+	var directories []string
 
 	for _, wdFile := range wdFiles {
 		// Add Dockerfiles in current directory, otherwise continue.
 		if !wdFile.IsDir() {
 			if wdFile.Name() == "Dockerfile" {
-				dockerfiles = append(dockerfiles, filepath.Join(".", wdFile.Name()))
+				directories = append(directories, ".")
 			}
 			continue
 		}
 
 		// Add Dockerfiles in one sub-directory below.
-		subFiles, err := afero.ReadDir(opts.fs, filepath.Join(".", wdFile.Name()))
+		subFiles, err := afero.ReadDir(opts.fs, wdFile.Name())
 		if err != nil {
 			return nil, fmt.Errorf("read directory: %w", err)
 		}
 		for _, f := range subFiles {
 			if f.Name() == "Dockerfile" {
-				dockerfiles = append(dockerfiles, filepath.Join(".", wdFile.Name(), "Dockerfile"))
+				directories = append(directories, "./"+wdFile.Name())
```

Comment:
Will this work on other OSes without `filepath.join`?

Reply:
Why not use filepath.join?

Reply:
Because [`filepath.Join`](https://play.golang.org/p/RnnewVzqoTm) doesn't return what we need. I'll look for an alternative that's platform independent.

Reply:
Update this flow to the following:
* for `Dockerfile`s in the "root" we append `filepath.Dir(wdFile.Name())` which would result in `"."`
* for `Dockerfile`s in one sub-directory down we simply append the directory name via `wdFile.Name()`

`docker build -t <some-tag> <dirname>` is perfectly valid.